Samples/BatchSupport/BatchSupport.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.1000
```

Comment:
I samples are at VS 2017

Reply:
Do I just need to update my version of Visual studio? I thought I was on the latest for Visual Studio 2019

Reply:
I'm fine with making this sample Visual Studio 2019 but then we'll need to update the README documentation of this sample to indicate that 2019 is needed.

Reply:
I'm fine with keeping it at VS 2017 to be consistent with the rest of the samples but I'm not sure how to edit this file. From what I understand this is solution file is generated and used by Visual Studio so it's not regularly editable. I figure I could edit in NotePad or a regular text editor but is that the best solution?

Reply:
Yeah just edit solution file.

Please keep the original : # Visual Studio 15
VisualStudioVersion = 15.0.28307.1000